eva/optimizer/optimizer_tasks.py

```diff
@@ -281,7 +282,10 @@ def execute(self):
         for child_id in self.root_expr.children:
             child_grp = memo.get_group_by_id(child_id)
             if child_grp.get_best_expr(PropertyType.DEFAULT):
-                cost += child_grp.get_best_expr_cost(PropertyType.DEFAULT)
+                # Note: May never get hit when using EVA on Ray
```

Comment:
Any reason for this comment?

Reply:
Jsut for future ref. Will remove it soon